.github/workflows/ci.yml

```diff
@@ -58,3 +58,17 @@ jobs:
         with:
           command: test
           args: --verbose
+
+  miri:
+    name: "Miri"
+    runs-on: ubuntu-latest
+    continue-on-error: true # Needed until all Miri errors are fixed
```

Comment:
Added Miri to the CI, however it still runs into some errors in the `MetaTable` code that is handling vtable pointers. So I set `continue-on-error: true` for now (hopefully this does what I think it does...)

Reply:
Well it seems like there is no way to not show the red `X` in the PR list: https://github.com/orgs/community/discussions/15452